client/build.gradle.kts

```diff
@@ -18,60 +24,15 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import com.google.protobuf.gradle.*
-import io.spine.generate.dart.Extension
-
-buildscript {
-
-    repositories {
-        mavenLocal()
-        jcenter()
-        maven {
-            setUrl("https://spine.mycloudrepo.io/public/repositories/releases")
-        }
-    }
-
-    dependencies {
-        classpath("io.spine.tools:spine-proto-dart-plugin:1.6.4")
-    }
-}
-
-plugins {
-    codegen
-}
-
-spine.enableJava()
+import com.google.protobuf.gradle.protobuf
+import io.spine.dart.gradle.Extension
 
-// For now, as Proto Dart plugin is not published to the Gradle plugin registry, apply it via
-// an `apply` statement instead of in the `plugins` section.
-apply(plugin = "io.spine.tools.proto-dart-plugin")
-
-tasks.assemble {
-    dependsOn("generateDart")
-}
-
-protobuf {
-    generateProtoTasks {
-        all().forEach { task ->
-            task.plugins {
-                id("dart")
-                remove("spineProtoc")
-            }
-            task.builtins {
-                remove("java")
-            }
-        }
-    }
-}
+spine.enableDart()
 
 dependencies {
     protobuf(project(":model"))
 }
 
-tasks.withType(JavaCompile::class) {
-    enabled = false
-}
-
 // Since Proto Dart plugin is not applied in the `plugins` section, Gradle does not generate kotlin DSL for it.
 // This will be fixed in future by allowing to configure Dart subprojects via Bootstrap. Meanwhile, we configure
 // the `protoDart` extension like this.
```

Comment:
Does this comment and the code still apply, assuming we have `spine.enableDart()` above?

Reply:
Yep. That's still relevant. We are not yet able to configure this through the `dart` section of the bootstrap plugin.